ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1165,3 +1165,4 @@ admin.message.queue.tab.scheduled = Scheduled
 admin.message.queue.tab.due = Due
 admin.message.queue.tab.muted.future = Will not send
 admin.message.queue.tab.muted.past = Did not send
+message.username.taken = User name '{{username}}' is already taken.
```

Comment:
Let's name this `username.taken`, since the other keys pertaining to usernames start with `username`.